opentracing/tracer.py

```diff
@@ -34,44 +33,53 @@ class Tracer(SpanPropagator):
 
     singleton_noop_span = Span()
 
-    def start_trace(self, operation_name=None, tags=None):
-        """Starts a new trace and creates a new root span.
+    def start_span(self, operation_name, parent=None, **kwargs):
```

Comment:
I'll change to named optional params because I don't feel strongly about this, but just FYI I meant that the kwargs version doesn't lock *us* into the particular order of the optional params... i.e., it's more forwards-compatible. My concern is that a user will omit the param names and then we add a new param anywhere other than the end of the param list. The underlying issue:

https://repl.it/BnRP/0